src/communication/request_client_server.hpp

```diff
@@ -238,7 +243,7 @@ namespace cubcomm
   }
 
   template <typename MsgId>
-  void request_server<MsgId>::register_request_handler (MsgId msgid, server_request_handler &handler)
+  void request_server<MsgId>::register_request_handler (MsgId msgid, const server_request_handler &handler)
```

Comment:
GCC complains in:

```
   template <typename ... PackableArgs>
   int request_client_server<ClientMsgId, ServerMsgId>::send (ClientMsgId msgid, const PackableArgs &... args)
   {
    return send_client_request (this->request_server<ServerMsgId>::m_channel, msgid, args...);
   }
```
wants to have `m_channel` qualified; otherwise it does not 'see' it

Reply:
fixed